nestkernel/event.h

```diff
@@ -258,6 +258,16 @@ class Event
    */
   virtual void set_diffusion_factor( weight t ){};
 
+  /**
+   * Returns true if the pointer to the sender node is not null.
+   */
+  bool sender_is_not_null() const;
```

Comment:
This is fine, but I wonder if we shouldn't use a positive name, e.g., `sender_valid()`, same for receiver.

Reply:
@heplesser My reasoning behind this name was to avoid having to use `not` everywhere these functions were used. But I agree that positive names look better, so I have changed them now.